lib/utils.pm

```diff
@@ -1492,7 +1492,7 @@ sub reconnect_mgmt_console {
     elsif (is_x86_64) {
         if (is_ipmi) {
             select_console 'sol', await_console => 0;
-            assert_screen([qw(qa-net-selection prague-pxe-menu grub2)], 300);
+            assert_screen([qw(qa-net-selection prague-pxe-menu grub2)], 300) unless get_var('IPXE');
             # boot to hard disk is default
             send_key 'ret';
```

Comment:
I think the `send_key` should be skipped together with the `assert_screen` above, since it's supposed to select the default item in the PXE menu.

Reply:
Of course, will change that